spm-monitor-utils/src/main/java/com/sematext/spm/client/MonitorUtil.java

```diff
@@ -697,4 +696,30 @@ public static String resolveHostnameInJava() {
       return "unknown";
     }
   }
+  
+  private static Map<String, Properties> LOADED_MONITOR_PROPERTIES = new HashMap<String, Properties>();
+  private static Map<String, Long> LAST_MONITOR_PROPERTIES_RELOAD_TIME = new HashMap<String, Long>();
+  private static long ONE_MINUTE_MS = 60 * 1000;
+  private static long MONITOR_PROPERTIES_RELOAD_INTERVAL = ONE_MINUTE_MS;
+
+  public static Properties loadMonitorProperties(File monitorPropertiesFile) {
```

Comment:
Could this method be called by multiple threads? we are sharing LOADED_MONITOR_PROPERTIES & LAST_MONITOR_PROPERTIES_RELOAD_TIME

Reply:
There is just a single thread, but I can make this synchronized, just in case we one day introduce multiple threads (e.g. multiple agents).